ppr-ui/src/composables/mhrInformation/useTransferOwners.ts

```diff
@@ -612,6 +599,15 @@ export const useTransferOwners = (enableAllActions: boolean = false) => {
     return currentGroup && (!isEqualNumerator || !isEqualDenominator)
   }
 
+  /** Return true if a member of a specified partyType has been added to the group **/
+  const hasAddedpartyTypeToGroup = (groupId: number, partyType: HomeOwnerPartyTypes): boolean => {
+    const value = getMhrTransferHomeOwnerGroups.value
+    return value
+      .find(group => group.groupId === groupId).owners
+      .some(owner => owner.partyType === partyType &&
+        owner.action === ActionTypes.ADDED)
+  }
```

Comment:
Nice work
You can probably use `getMhrTransferHomeOwnerGroups.value` directly considering it's only accessed once.
Your current spacing also accommodates this just fine i think 👍